locale/de.po

```diff
@@ -1000,10 +998,10 @@ msgid "Channel selection settings"
 msgstr "Senderliste-Einstellungen"
 
 msgid "Show in Movie Center?"
-msgstr "In der Filmauswahl anzeigen?"
+msgstr "In der Filmauswahl anzeigen"
```

Comment:
Hi @Makumbo 
diese ? und die anderen auch wieder zurückdrehen.
Bitte!!

Reply:
Wie meinst Du das mit 'zurückdrehen'?

Fragezeichen (oder Doppelpunkte) gehören **nicht** ans Ende von Menüpunkten.
"In der Filmauswahl anzeigen" kann nur mit "Ja" oder "Nein" beantwortet werden, ist an sich also schon eine Frage.
Aber, wie auch bei Menüpunkten im PC braucht es keine Fragezeichen, ja nicht mal Punkte am Ende, was eigentlich richtig wäre.
Schau' Dir Deine PC Menüpunkte mal an...